airbyte-integrations/connectors/source-plaid/source_plaid/manifest.yaml

```diff
@@ -108,6 +108,9 @@ streams:
       - type: RemoveFields
         field_pointers:
           - - type
+      - type: RemoveFields
+        field_pointers:
+          - - transaction_type
```

Comment:
This is in the `balances` stream definition. Could you move it down to the `transaction` stream?

I also wonder about the double dash since the [example](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/record-selector#removing-fields) shows one dash and an array, but I'll defer to you.